--- mate-applets-1.2.3/mini-commander/src/preferences.c~	2012-05-28 20:01:00.000000000 +0000
+++ mate-applets-1.2.3/mini-commander/src/preferences.c	2012-06-11 17:15:24.895422399 +0000
@@ -415,7 +415,7 @@
 					           GTK_DIALOG_DESTROY_WITH_PARENT,
 					           GTK_MESSAGE_ERROR,
 					           GTK_BUTTONS_OK,
-					           error_message);
+					           "%s", error_message);
 
 	    g_signal_connect (error_dialog, "response", G_CALLBACK (gtk_widget_destroy), NULL);
 	    gtk_window_set_resizable (GTK_WINDOW (error_dialog), FALSE);
